storage/core/src/server/Helpers.ts

```diff
@@ -127,3 +128,7 @@ export async function uploadToUrl(
     headers,
   });
 }
+
+export function getRandomString(): string {
```

Comment:
Should add comment that this should be replaced by randomUUID() once we drop support for node.js 12/14.